docs/classes/_decoder_.decoder.md

```diff
@@ -484,7 +485,7 @@ ___
 
 ### `<Static>` dict
 
-▸ **dict**A(decoder: *[Decoder](_decoder_.decoder.md)<`A`>*): [Decoder](_decoder_.decoder.md)<`object`>
+▸ **dict**A(decoder: *[Decoder](_decoder_.decoder.md)<`A`>*): [Decoder](_decoder_.decoder.md)<`Record`<`string`, `A`>>
```

Comment:
I don't think my changes should have affected this; was it just out of date?

Reply:
Oh ya, don't worry about that. I guess I didn't update the docs after my last change.